src/daemon/daemon_init_settings.cpp

```diff
@@ -53,6 +53,14 @@ QString persistent_settings_filename()
 
 QString driver_interpreter(QString val)
 {
+    val = val.toLower();
+
+    if (val == QStringLiteral("hyper-v"))
+        val = QStringLiteral("hyperv");
+
+    if (val == QStringLiteral("vbox"))
```

Comment:
Small detail that later occurred to me:

```suggestion
    else if (val == QStringLiteral("vbox"))
```